ui/base/src/main/res/values/strings_base_ui.xml

```diff
@@ -5,6 +5,7 @@
 
     <!-- Tool Categories -->
     <eat-comment />
+    <string name="tool_category_all_tools">All Tools</string>
```

Comment:
don't add this as a global category, put the "All Tools" string with the other dashboard tools strings